pkg/attribute/rootBag.go

```diff
@@ -304,52 +309,89 @@ func (rb *rootBag) update(dictionary dictionary, attrs *mixerpb.Attributes) erro
 		return err
 	}
 
+	log := glog.V(2)
```

Comment:
curious: would it be worth putting all of the log statements into a buffer and then issuing a single log statement so that all of the attribute bag updates would be in a single log line?

that way, we could avoid interweaving of updates in the logs across requests?

might make it easier to consume.